### ISSUE TITLE
Remove the ".js" from the imports

### ISSUE THREAD

**Issue by almet** [Kinto/kinto-client#63]
Some imports are inconsistent, and use ".js" in the module name. e.g:

```es6
import { partition, pMap, omit } from "./utils.js";
import HTTP from "./http.js";
```

Instead, we should remove the extraneous `.js` part:

```es6
import { partition, pMap, omit } from "./utils";
import HTTP from "./http";
```

@vsham20 would you like to work on this? It's an easy thing to do to get started! (if you do, please assign yourself in here), thanks!

**Comment by vsham20** [comment]
almet Sure, Could you please explain what exactly is this bug?

**Comment by vsham20** [comment]
almet: I can't assign it to myself, maybe permission issues?

**Comment by almet** [comment]
I've just assigned it to you. The issue is about changing the imports (removing the `.js` extension)

**Comment by vsham20** [comment]
@almet cool, in which file do i need to make these changes?

**Comment by almet** [comment]
@vsham20 you need to have a look at all the files of the project to find them. Here is what a quick search returns (I'm using [grin](https://pypi.python.org/pypi/grin) for this):

```bash

$ grin '.js"' src/
src/http.js:
    3 : import ERROR_CODES from "./errors.js";
src/index.js:
    6 : import { partition, pMap, omit } from "./utils.js";
    7 : import HTTP from "./http.js";
```

**Comment by n1k0** [comment]
Fixed by #67.
